clients/geth/specular/data/base_genesis.json

```diff
@@ -13,15 +13,44 @@
   "difficulty": "1",
   "gasLimit": "8000000",
   "extradata": "",
-  "alloc": {
-    "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266": {
+  "preDeploy": [
+    {
+      "address": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
```

Comment:
I'm not sure if this is better. To me using address as key helps to prevent duplicate pre-deploy on a single address when manually configuring

Reply:
remember to remove this since we have a copy in `config/`